### ISSUE TITLE
No image alternative text for folder and document types image 

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#378]
To comply to WCAG 2.0 AA standards all folder and document types images should have alternative text
![screen shot 2016-07-06 at 16 33 17](https://cloud.githubusercontent.com/assets/13200338/16623896/940038a2-4397-11e6-8102-4b49958dfb28.png)

**Comment by VitoAlbano** [comment]
Retested : 

all folders and documents have alternative text but for the folders it is always "Folder" and the files the type of the file (ex : "Image File") is it this correct? @hashraf88 
<img width="654" alt="screen shot 2016-07-21 at 11 01 57" src="https://cloud.githubusercontent.com/assets/7974125/17019086/a2a90504-4f32-11e6-97b9-762a4f5d5f12.png">
<img width="441" alt="screen shot 2016-07-21 at 11 02 22" src="https://cloud.githubusercontent.com/assets/7974125/17019087/a2b07190-4f32-11e6-83cf-63454bdb7def.png">

**Comment by hashraf88** [comment]
@VitoAlbano yep that is fine, the alternative text should be of the image and nothing else.
